### ISSUE TITLE
'helperText' on 'TextField' causes error on `TextInput`

### ISSUE THREAD

**Issue by jomarko** [aerogear/uniforms-patternfly#106]
In our project, we are using '[uniforms](https://uniforms.tools/docs/api-forms/)' for generating forms in combination with 'patternfly-uniform' fields, especially '[TextField](https://github.com/aerogear/uniforms-patternfly/blob/master/src/TextField.tsx)'.

`TextField` delegates rendering to other React components, especially to `@patternfly/react-core TextInput`, what is fine, however the `TextInput` doesn't support `helperText` and throws attache error into browser console.

As the `uniforms-patternfly` already render `helperText` thanks to '[wrapField](https://github.com/aerogear/uniforms-patternfly/blob/master/src/wrapField.tsx) the solution should be simply not pass `helperText` down to `TextInput`.

```
console.error
    Warning: React does not recognize the `helperText` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `helpertext` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
        at input
        at TextInputBase (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/components/TextInput/TextInput.tsx:[9](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/runs/8096562216?check_suite_focus=true#step:7:10)9:5)
        at TextInput
        at div
        at div
        at FormGroup (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/components/Form/FormGroup.tsx:47:3)
        at Text (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/uniforms-patternfly/dist/es6/TextField.js:37:15)
        at Field (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/uniforms/cjs/connectField.js:39:65)
        at AutoField (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/uniforms/cjs/createAutoField.js:[12](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/runs/8096562216?check_suite_focus=true#step:7:13):64)
        at section
        at _ (/home/runner/work/sandbox-ui/sandbox-ui/src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx:88:33)
        at ConfigurationForm (/home/runner/work/sandbox-ui/sandbox-ui/src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx:21:5)
        at ConfigurationEdit (/home/runner/work/sandbox-ui/sandbox-ui/src/app/Processor/ProcessorEdit/ConfigurationEdit/ConfigurationEdit.tsx:21:5)
        at section
        at FormSection (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/components/Form/FormSection.tsx:17:3)
        at form
        at Form (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/components/Form/Form.tsx:17:3)
        at div
        at FlexItem (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/layouts/Flex/FlexItem.tsx:143:3)
        at div
        at Flex (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/layouts/Flex/Flex.tsx:342:3)
        at div
        at Flex (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/layouts/Flex/Flex.tsx:342:3)
        at div
        at Flex (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/layouts/Flex/Flex.tsx:342:3)
        at section
        at section
        at PageSection (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@patternfly/react-core/src/components/Page/PageSection.tsx:73:3)
        at ProcessorEdit (/home/runner/work/sandbox-ui/sandbox-ui/src/app/Processor/ProcessorEdit/ProcessorEdit.tsx:88:5)
        at Suspense
        at I18nextProvider (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/react-i18next/dist/commonjs/I18nextProvider.js:[13](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/runs/8096562216?check_suite_focus=true#step:7:14):19)
        at Object.<anonymous>.exports.I[18](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/runs/8096562216?check_suite_focus=true#step:7:19)nProvider (/home/runner/work/sandbox-ui/sandbox-ui/node_modules/@rhoas/app-services-ui-components/src/I18n/I18nProvider.tsx:[21](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/runs/8096562216?check_suite_focus=true#step:7:22):[26](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/runs/8096562216?check_suite_focus=true#step:7:27))
        at Providers (/home/runner/work/sandbox-ui/sandbox-ui/src/utils/testUtils.tsx:12:[41](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/runs/8096562216?check_suite_focus=true#step:7:42))
```

**Comment by wtrocki** [comment]
That seems like trivial fix. Feel free to make contribution for this.

**Comment by jomarko** [comment]
Working on this right now

**Comment by jomarko** [comment]
https://github.com/aerogear/uniforms-patternfly/pull/107

**Comment by jomarko** [comment]
Solved with https://github.com/aerogear/uniforms-patternfly/pull/107

Released as 4.7.6
